### PR TITLE
Correct keyword spacing

### PR DESCRIPTION
**Pull request by Nihlus**
This PR corrects the keyword spacing in the codebase, and enforces it in the future by enabling SA1000 from StyleCop.

### CODE REVIEW

File: src/OpenTK/Platform/Windows/WinRawKeyboard.cs [hunk 1]
```diff
@@ -107,7 +107,7 @@ public void RefreshDevices()
                         string deviceClassGUID = (string)regkey.GetValue("ClassGUID"); // for windows 8 support via OpenTK issue 3198
 
                         // making a guess at backwards compatability. Not sure what older windows returns in these cases...
-                        if(deviceClass == null || deviceClass.Equals(string.Empty)){
+                        if (deviceClass == null || deviceClass.Equals(string.Empty)){
```
Comment (Frassle) [inline src/OpenTK/Platform/Windows/WinRawKeyboard.cs]: Probably a separate PR but can we do something about braces at end of lines like this one?

**Comment by Frassle** [comment]
LGTM

File: src/OpenTK/Platform/Windows/WinRawKeyboard.cs (hunk 1 above)
Comment (Nihlus) [inline src/OpenTK/Platform/Windows/WinRawKeyboard.cs]: Yes, that's a separate StyleCop rule :)

**Comment by amulware** [comment]
Looks fine to me too. :+1:

**Comment by Nihlus** [comment]
:+1: I'll merge it in as soon as our PR density is a little lower. I'd rather not a cosmetic change affect an outstanding PR :)

**Pull request merged**